src/crypto/index.js

```diff
@@ -373,7 +378,7 @@ Crypto.prototype.isKeyBackupTrusted = async function(backupInfo) {
                 );
                 sigInfo.valid = true;
             } catch (e) {
-                logger.info("Bad signature from device " + device.deviceId, e);
+                logger.info("Bad signature from key ID " + keyId, e);
```

Comment:
Would it be useful to keep logging the `device.deviceId` and then also add the key ID?

Reply:
The device ID we just get by `keyId.split(':')[1]` so probably not really

Reply:
Okay, makes sense then!